### PR TITLE
initial commit and file structure

### PR DESCRIPTION
**Pull request by piknikki**
## What does this PR do?
 - I created a Vue project (including router and store (Vuex), as well as Cypress), and then changed the logo to our logo and removed the junk we don't need.
## How should it be tested?
 - n/a -- just make sure you check out the file structure

### CODE REVIEW

**Comment by jequeza** [comment]
Very interesting stuff devs, great work. I'm loving it!!! 🦑

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+npm run test:e2e
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
```
Comment (m1073496) [inline README.md:29]: Did we add a second README with Vue or do these changes apply to the original README?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    '@vue/standard'
+  ],
+  parserOptions: {
+    parser: 'babel-eslint'
+  },
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off'
+  }
+}
```
Comment (knees4bees) [inline .eslintrc.js:17]: Looks like we'll want to add our own rules to this `.eslintrc` with whatever rules we want. Could snag the rules from the `eslintrc` we used in mod 2?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title><%= htmlWebpackPlugin.options.title %></title>
```
Comment (knees4bees) [inline public/index.html:8]: Is this a dynamic way to display the title of the app on the browser tab?? Very cool. I've always updated this manually.

**Review by knees4bees — APPROVED** [review]
Awesome, Nikki! Thanks for getting the ball rolling here.

File: README.md (hunk 1 above)
Comment (piknikki) [reply README.md:29]: Yeah, the second one over-wrote the original.

File: .eslintrc.js (hunk 2 above)
Comment (piknikki) [reply .eslintrc.js:17]: this project is using a standard eslinter, so it's pretty much already set up.

Comment (piknikki) [reply .eslintrc.js:17]: What rules did you want to add, though?

File: public/index.html (hunk 3 above)
Comment (piknikki) [reply public/index.html:8]: Yep, that's how it gets set up through the vue cli. Didn't create-react-app do that, too? I can't remember.

File: .eslintrc.js (hunk 2 above)
Comment (knees4bees) [reply .eslintrc.js:17]: Oh maybe I'm confused. I thought it didn't really have any rules yet. I was thinking about the usual things like indentation, etc.

File: public/index.html (hunk 3 above)
Comment (knees4bees) [reply public/index.html:8]: Nice. With `create-react-app` it just ended up with `<title>React App</title>` or something like that and you had to manually change it to `<title>My Awesome App Name</title>`.

**Pull request merged**